pkg/unique-id/module.go

```diff
@@ -0,0 +1,126 @@
+package unique_id
+
+import (
+	"context"
+	"fmt"
+	"sync"
+
+	"github.com/applike/gosoline/pkg/cfg"
+	"github.com/applike/gosoline/pkg/kernel"
+	"github.com/applike/gosoline/pkg/log"
+)
+
+type uniqueIdGenerator struct {
+	kernel.EssentialBackgroundModule
+	kernel.EssentialStage
```

Comment:
So your module should not write metrics because the metric daemon is also in this stage

Reply:
Indeed, and luckily it's not

Reply:
So ddb writes no metrics? Okay, that is good, wasn't sure about that